### DISCUSSION TITLE
add an "are you sure" prompt that shows when you click retry

### DISCUSSION THREAD

**Discussion by minihoot** [ppy/osu#25771] [Ideas]
i was playing osu!lazer and i accidentally pressed retry and lost my progress. i think an "are you sure" prompt would fix this.

**Comment by peppy** [comment]
We could probably make these dangerous buttons, requiring a hold to quit/retry. But I think that might also not be what a lot of people want..

**Reply by bdach** [reply]
If I were to entertain this proposal it would need to respect the hold activation time in settings for the sake of players that don't want it.

**Reply by minihoot** [reply]
bdach's idea is pretty good
